openedx/core/djangoapps/user_api/urls.py

```diff
@@ -9,6 +9,7 @@
 from .accounts.views import AccountDeactivationView, AccountViewSet
 from .preferences.views import PreferencesDetailView, PreferencesView
 from .verification_api.views import PhotoVerificationStatusView
+from lms.djangoapps.sga_api.views import CreateUserAccountWithoutPasswordView, BulkEnrollView
```

Comment:
It is a wrong place for the package import. And also a wrong place for the new api endpoint urls file.
Move it to the `sga_api` dir.